skimage/graph/_mcp.pyx

```diff
@@ -391,9 +391,13 @@ cdef class MCP:
         """
         pass
 
-
+    @deprecate_parameter(
+        "max_cumulative_cost",
+        start_version="0.25",
+        stop_version="0.27",
+    )
     def find_costs(self, starts, ends=None, find_all_ends=True,
-                   max_coverage=1.0, max_cumulative_cost=None, max_cost=None):
+                   max_coverage=1.0, max_cumulative_cost=DEPRECATED, max_step_cost=None):
```

Comment:
Technically, we should also deprecate `max_cost` if we rename refactor it. Just a simple `@deprecate_parameter` pointing at `max_step_cost`. 

I'm happy to do push these things myself in case you don't have the time or are not clear what I mean here. ;)

Reply:
I also deprecated `max_cost`.

Reply:
Thank you. :blush: